tests/observability/runbook_url/test_runbook_url.py

```diff
@@ -77,10 +77,23 @@ def test_runbook_downstream_urls(cnv_prometheus_rules_unique_alert_names_runbook
         if error:
             error_messages.append(error)
     if alerts_without_runbook:
-        LOGGER.error(f"Runbook url missing for following CNV alerts: {alerts_without_runbook}")
-        raise AssertionError("CNV alerts with missing runbook url found.")
+        LOGGER.error(f"Runbook url missing for following CNV runbook_url: {alerts_without_runbook}")
```

Comment:
Each `alert` must have a `runbook_url`, so do we need to change the wording here?